spec/controllers/movies_controller_spec.rb

```diff
@@ -0,0 +1,12 @@
+require 'rails_helper'
+
+RSpec.describe MoviesController, type: :controller do
```

Comment:
As far as I'm aware, we don't typically do controller specs, but left it here since it was auto-generated.